experiments/chaos/openebs_target_failure/test_prerequisites.yml

```diff
@@ -86,7 +86,7 @@
 
   - name: Record the storage engine name
     set_fact:
-      stg_engine: "{{ openebs_stg_engine.stdout }}"
+      stg_engine: "jiva-csi"
```

Comment:
@payes if we didn't get the stg_engine name from storage class the above two task is not needed

Reply:
removed